boxoffice/models/item.py

```diff
@@ -79,12 +91,79 @@ def quantity_available(self):
 
     @property
     def is_available(self):
-        """Checks if an item has a current price object and has a positive quantity_available"""
+        """Check if an item has a current price object and has a positive quantity_available"""
         return bool(self.current_price() and self.quantity_available > 0)
 
     def is_cancellable(self):
         return datetime.utcnow() < self.cancellable_until if self.cancellable_until else True
 
+    def url_for(self, action='view', _external=False, **kwargs):
+        if action == 'view':
+            return url_for('admin_item', item_id=self.id, _external=_external, **kwargs)
+
+    @property
+    def url_for_view(self):
+        return self.url_for('view')
```

Comment:
Is this necessary? Using `item.url_for()` should suffice in templates, since 'view' is the default action.